core/kernel.rbs

```diff
@@ -655,14 +658,19 @@ module Kernel : BasicObject
   #
   def self?.Rational: (int | _ToR rational_like, ?exception: true) -> Rational
                     | (int | _ToR rational_like, exception: bool) -> Rational?
-                    | (int | _ToR numer, ?int | _ToR denom, ?exception: true) -> Rational
-                    | (int | _ToR numer, ?int | _ToR denom, exception: bool) -> Rational?
-                    | [T] (Numeric & _RationalDiv[T] numer, Numeric denom, ?exception: bool) -> T
+                    | (int | _ToR numer, int | _ToR denom, ?exception: true) -> Rational
+                    | (int | _ToR numer, int | _ToR denom, exception: bool) -> Rational?
```

Comment:
I simplified the signature of this a bit, collapsing the ` int | _ToR numer, ?...` into the ones above it